tests/playground.html

```diff
@@ -1460,7 +1610,7 @@ <h1>Blockly Playground</h1>
         </statement>
       </block>
     </category>
-    <category name="Fields" expanded="true">
+    <category name="Fields">
```

Comment:
I think you need to leave ``expanded="true"`` to make sure it's expanded by default.